api/controllers/banis.js

```diff
@@ -12,7 +13,15 @@ const lengthExistsMap = {
 const pool = createPool(config.mysql);
 
 const error = (err, res) => {
-  res.status(400).json({ error: true, data: err });
+  console.error(err);
+  Error.captureStackTrace(err);
```

Comment:
I purposely avoided expanding the scope of this PR. This will be done in a future PR for #96 and #108. I have laid the groundwork for it here because I needed it for this PR.